marklogic-data-hub/src/test/java/com/marklogic/hub/job/JobDataManagerTest.java

```diff
@@ -0,0 +1,56 @@
+package com.marklogic.hub.job;
+
+import com.marklogic.client.io.DocumentMetadataHandle;
+import com.marklogic.hub.AbstractHubCoreTest;
+import com.marklogic.hub.DatabaseKind;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static com.marklogic.client.io.DocumentMetadataHandle.Capability.*;
+import static com.marklogic.client.io.DocumentMetadataHandle.Capability.EXECUTE;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class JobDataManagerTest extends AbstractHubCoreTest {
+
+    @BeforeEach
+    public void setup(){
+        runAsAdmin();
+        clearDatabase(getHubClient().getStagingClient());
```

Comment:
No need to do these 3 lines of code, the parent class does this for you automatically